kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/user/ServiceUserRepository.java

```diff
@@ -61,6 +61,7 @@
   private static final RequestBody EMPTY_BODY =
       RequestBody.create("", MediaType.parse("application/json; charset=utf-8"));
   private static final Duration FETCH_THRESHOLD = Duration.ofMinutes(1L);
+  private static final Duration CONNECTION_TIMEOUT = Duration.ofSeconds(30);
 
```

Comment:
Can you create a READ_TIMEOUT value too? Just to avoid confusion and change these values separately in the future.

Reply:
I also increased it to 60 seconds because I was still getting the timeout exception after a while (although less). Is that ok?

Reply:
Yes good. We have a lot of users on prod so can be expected.